src/services/formatting/formatting.ts

```diff
@@ -34,7 +34,7 @@ namespace ts.formatting {
      * the first token in line so it should be indented
      */
     interface DynamicIndentation {
-        getIndentationForToken(tokenLine: number, tokenKind: SyntaxKind, container: Node): number;
+        getIndentationForToken(tokenLine: number, tokenKind: SyntaxKind, container: Node, suppressDelta?: boolean): number;

```

Comment:
Since this is internal there's no need to be backwards-compatible. So this should just be `suppressDelta: boolean`.